flink-java/src/test/java/org/apache/flink/api/java/io/CsvInputFormatTest.java

```diff
@@ -984,7 +984,7 @@ public void testPojoTypeWithInvalidFieldMapping() throws Exception {
 	@Test
 	public void testQuotedStringParsingWithIncludeFields() throws Exception {
 		final String fileContent = "\"20:41:52-1-3-2015\"|\"Re: Taskmanager memory error in Eclipse\"|" +
-				"\"Blahblah <blah@blahblah.org>\"|\"bla\"|\"blubb\"";
```

Comment:
Let's not change old checks, but rather add new checks. You never know if you don't loose coverage of a critical case otherwise...